osmaxx/excerptexport/models/export.py

```diff
@@ -65,3 +76,23 @@ def _get_export_status_changed_message(self):
             'job_progress/messages/export_status_changed.txt',
             context=view_context,
         ).strip()
+
+    def _fetch_result_file(self):
+        from osmaxx.api_client import ConversionApiClient
+        from . import OutputFile
+        api_client = ConversionApiClient()
+        file_content = api_client.get_result_file(self.conversion_service_job_id)
+        file_location = get_default_private_storage().save(
+            name=str(uuid.uuid4()),
```

Comment:
This should probably be a better name than this, something with an UUID inside it's name would be great, but also some more information like the file_format, its name (from the order) maybe more?